docs/src/main/sphinx/connector/hive.rst

```diff
@@ -17,6 +17,7 @@ Hive connector
     IBM Cloud Object Storage <hive-cos>
     Storage Caching <hive-caching>
     Alluxio <hive-alluxio>
+    Object storage file formats <object-storage-file-formats>
```

Comment:
I guess this is fine for now .. but needs to move outside of the hive context over time

Reply:
Yes. Phase 4 is re-org.